### PR TITLE
DEPS: unpin Matplotlib

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

**Comment by jimustafa** [comment]
I think the matplotlib version _should_ be pinned and match the version tag that is shown on the cheatsheets. That way, the cheatsheets will be built with a definite version of matplotlib. The matplotlib version can be bumped, along with version tag on cheatsheets, using the `bump2version` utility.

Yes, `make check` fails because the installed matplotlib version does not match the cheatsheets version tag.

**Comment by tacaswell** [comment]
Fair enough.

I also added a section to the read me documenting how to use `bump2version`.

File: README.md [hunk 1]
```diff
@@ -60,3 +60,24 @@ $ cd ..
 $ xelatex cheatsheets.tex
 $ xelatex cheatsheets.tex
 ```
+
+## Update Matplotlib version
+
+The version of Matplotlib is hard-coded a number of places throughout the
```
Comment (jimustafa) [inline README.md:66]: hard-coded _in_ a number of places...

File: README.md [hunk 2]
```diff
@@ -60,3 +60,24 @@ $ cd ..
...
+cheatsheets.  To ensure that they are all consistent (and consistent with the
+version of Matplotlib actually used) use the tool `bump2version` to update.
+For patch releases do
+
+```
+$ bump2release --new-version x.y.Z patch
```
Comment (jimustafa) [inline README.md:72]: should this just be `bump2version patch`

File: README.md [hunk 3]
```diff
@@ -60,3 +60,24 @@ $ cd ..
...
+$ cd requirements
+$ pip-compile requirements.in > requirements.txt
+```
+
+and for a new minor versions do
+
+```
+$ bump2release --new-version x.Y.0 minor
```
Comment (jimustafa) [inline README.md:80]: should this just be `bump2version minor`

**Review by jimustafa — CHANGES_REQUESTED** [review]

**Comment by dstansby** [comment]
I'm going to close this in favour of other more recent PRs to bump the MPL version.

**Pull request closed without merging**